### PR TITLE
docs(plugin): refactor proxy-mirror.md

### PR DESCRIPTION
**Pull request by lunarwhite**
### Description

update documents under `docs/zh/latest/plugins/` along with `docs/en/latest/plugins/`

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/plugins/proxy-mirror.md [hunk 1]
```diff
@@ -59,29 +67,60 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f1
 }'
 ```
 
-测试：
+### 指定 mirror 子请求的超时时间
+
+我们可以在 `conf/config.yaml` 的 `plugin_attr` 中指定子请求的超时时间。这在连接复用的场景下 mirror 流量到一个非常慢的后端服务时非常有用。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/proxy-mirror.md:72]:
```suggestion
我们可以在 `conf/config.yaml` 文件内的 `plugin_attr` 中指定子请求的超时时间。由于镜像请求是以子请求的方式实现，子请求的延迟将会导致原始请求阻塞，直到子请求完成，才可以恢复正常。因此可以配置超时时间，来避免子请求出现过大的延迟而影响原始请求。
```

File: docs/zh/latest/plugins/proxy-mirror.md [hunk 2]
```diff
@@ -59,29 +67,60 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f1
 }'
 ```
 
-测试：
+### 指定 mirror 子请求的超时时间
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/proxy-mirror.md:70]:
```suggestion
### 指定镜像子请求的超时时间
```

File: docs/zh/latest/plugins/proxy-mirror.md [hunk 3]
```diff
@@ -59,29 +67,60 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f1
...
+
+我们可以在 `conf/config.yaml` 的 `plugin_attr` 中指定子请求的超时时间。这在连接复用的场景下 mirror 流量到一个非常慢的后端服务时非常有用。
+
+| 名称 | 类型 | 默认值 | 描述 |
+| --- | --- | --- | --- |
+| connect | string | 60s | 镜像请求到上游的连接超时时间。 |
+| read | string | 60s | 镜像请求到上游的读超时时间。 |
+| send | string | 60s | 镜像请求到上游的写超时时间。 |
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/proxy-mirror.md:78]:
```suggestion
| send | string | 60s | 镜像请求发送到上游的超时时间。 |
```

File: docs/zh/latest/plugins/proxy-mirror.md [hunk 4]
```diff
@@ -23,26 +28,29 @@ title: proxy-mirror
 
 ## 描述
 
-代理镜像插件，该插件提供了镜像客户端请求的能力。
+`proxy-mirror` 插件提供了镜像客户端请求的能力。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/proxy-mirror.md:31]:
```suggestion
`proxy-mirror` 插件提供了镜像客户端请求的能力。流量镜像是将线上真实流量拷贝到镜像服务中，以便在不影响线上服务的情况下，对线上流量或请求内容进行具体的分析。
```

File: docs/en/latest/plugins/proxy-mirror.md [hunk 5]
```diff
@@ -83,26 +83,27 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f1
...
-Content-Length: 12
-Connection: keep-alive
-Server: APISIX web server
-Date: Wed, 18 Mar 2020 13:01:11 GMT
-Last-Modified: Thu, 20 Feb 2020 14:21:41 GMT
+| Name | Type | Default | Description |
+| --- | --- | --- | --- |
+| connect | string | 60s | Connection timeout for mirror request to upstream. |
+| read | string | 60s | Reading timeout for mirror request to upstream. |
+| send | string | 60s | Sending timeout for mirror requests to upstream. |
```
Comment (SylviaBABY) [inline docs/en/latest/plugins/proxy-mirror.md:94]:
```suggestion
| send | string | 60s | The time that APISIX maintains the connection with the mirror server; if APISIX does not send a request within this time, the connection is closed. |
```

File: docs/zh/latest/plugins/proxy-mirror.md [hunk 6]
```diff
@@ -59,29 +67,60 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f1
...
+      send: 2000ms
+```
+
+## 测试插件
+
+:::tip 提示
+
+因为指定的镜像地址是 `127.0.0.1:9797`，所以验证此插件是否正常工作需要在端口为 `9797` 的服务上确认。
+
+我们可以通过 `python` 启动一个简单的 server：
```
[the remark this answers is not in the document]
Comment (hf400159) [reply docs/zh/latest/plugins/proxy-mirror.md:95]: I'm not sure if this partial operation is required. The steps for each user to start the service are different.

File: docs/zh/latest/plugins/proxy-mirror.md (hunk 1 above)
Comment (lunarwhite) [reply docs/zh/latest/plugins/proxy-mirror.md:72]: 👍

File: docs/zh/latest/plugins/proxy-mirror.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (lunarwhite) [reply docs/zh/latest/plugins/proxy-mirror.md:95]: What's your suggestion? Shall we remove this step?

**Review by SylviaBABY — DISMISSED** [review]

File: docs/zh/latest/plugins/proxy-mirror.md [hunk 7]
```diff
@@ -59,29 +67,60 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f1
...
 
-测试：
+### 指定镜像子请求的超时时间
+
+我们可以在 `conf/config.yaml` 文件内的 `plugin_attr` 中指定子请求的超时时间。由于镜像请求是以子请求的方式实现，子请求的延迟将会导致原始请求阻塞，直到子请求完成，才可以恢复正常。因此可以配置超时时间，来避免子请求出现过大的延迟而影响原始请求。
+
+| 名称 | 类型 | 默认值 | 描述 |
+| --- | --- | --- | --- |
+| connect | string | 60s | 镜像请求到上游的连接超时时间。 |
+| read | string | 60s | 镜像请求到上游的读超时时间。 |
```
Comment (tzssangglass) [inline docs/zh/latest/plugins/proxy-mirror.md:77]:
```suggestion
| read | string | 60s | APISIX 与镜像服务器维持连接的时间；如果在该时间内，APISIX 没有收到镜像服务器的响应，则关闭连接。 |
```

Comment (tzssangglass) [reply docs/zh/latest/plugins/proxy-mirror.md:77]: also change the en

**Review by tzssangglass — APPROVED** [review]

**Review by SylviaBABY — APPROVED** [review]

**Pull request merged**